File: class01/cpp/AP0101-MediaDuasNotas.cpp
#include <iostream>
#include <iomanip>
int main() {
    double nota1, nota2;
    std::cout << "Digite a primeira nota: ";
    std::cin >> nota1;
    std::cout << "Digite a segunda nota: ";
    std::cin >> nota2;
    double media = (nota1 + nota2) / 2;
    std::cout << std::fixed << std::setprecision(1) << "Media: " << media << std::endl;
    std::cout << (media >= 7 ? "Aprovado" : "Reprovado") << std::endl;
    return 0;
}

File: class01/java/AP0101-MediaDuasNotas.java
import java.util.Scanner;

public class MediaDuasNotas {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite a primeira nota: ");
        double nota1 = sc.nextDouble();
        System.out.print("Digite a segunda nota: ");
        double nota2 = sc.nextDouble();
        double media = (nota1 + nota2) / 2;
        System.out.printf("Media: %.1f%n", media);
        System.out.println(media >= 7 ? "Aprovado" : "Reprovado");
    }
}
